Dynasty Warriors 3: Achievement description cleanup

Argenrost opened the pull request:
Wanted to start by saying I really appreciate the work you put into this, vanilla DW3 was also a big part of my younger years and continues to influence my love for the series to this day.

I don't have the time or the know-how just yet to contribute actual achievements, but I wanted to propose some small updates to the titles and descriptions if possible. A lot of these changes are just minor bits of text and punctuation cleanup, but there was also a small error where the achievements for clearing 7-stage and 10-stage Musou Modes were given the wrong set of caveats in their descriptions (7-stage Musou Modes were listed as leader required, 10-stage were non-leader).

Cheers!

Argenrost committed:
Update Dynasty_Warriors_3.rascript

Corrected several achievement descriptions and titles with better formatting and proper naming conventions.

Argenrost committed:
Update Dynasty_Warriors_3.rascript

Extra periods not necessary in this context.

Kinglink commented on Dynasty_Warriors_3_Playstation_2/Dynasty_Warriors_3.rascript:
```diff
@@ -4,7 +4,7 @@
 
 onePlayerString = " [1P]"
-asASinglePlayerString = " as a single player"
+asASinglePlayerString = " as a single player."
```
Can you please remove the punctuation changes (At least the periods at the end of Descriptions).  I appreciate the thought, but it's not required (as long as it's uniformed) and not prefered. 

Reference:
https://docs.retroachievements.org/guidelines/content/writing-policy.html#punctuation-1

Argenrost replied:
Done, feel free to look it back over and see if I missed anything.

Kinglink commented on Dynasty_Warriors_3_Playstation_2/Dynasty_Warriors_3.rascript:
```diff
@@ -4413,7 +4413,7 @@ achievement("Diao Chan Eclipses the Moon",
 // Meng Huo was famously captured by Zhuge Liang during the Southern Campaign (Battle of Nanman) Seven Times.
 achievement("Nanman Jackpot",
-            "Defeat exactly 7 Officers as Meng Huo during Battle of Nanman with the Nanman Forces"  + asASinglePlayerString + " (Kills must be done by the player, bodyguard and other kills will not count)", 
+            "Defeat exactly 7 officers as Meng Huo during The Nanman Campaign with the Nanman Forces"  + asASinglePlayerString + " (Kills must be done by the player, bodyguard and other kills will not count.)", 
```
Good catch.

Kinglink reviewed (changes requested):
Excellent changes, please fix the punctuation.

Kinglink reviewed (approved):
This commit looks fine.

Kinglink reviewed (changes requested):
This commit looks fine. (Though please revert the other punctuation changes.

Argenrost committed:
Update Dynasty_Warriors_3.rascript: Requested review changes

Removed periods throughout the initial changelist to satisfy writing requirements.

Argenrost reviewed (commented):
Updated in accordance with the punctuation guidelines. I tried to remove as many periods as possible while keeping it looking clean; if descriptions shouldn't be ended with periods I'd rather avoid them entirely to keep it from looking awkward.

Argenrost committed:
Update Dynasty_Warriors_3.rascript: Additional corrections

Removed more stray periods and made some additional formatting improvements in a second pass.

Argenrost reviewed (commented):
I did a second pass out of curiosity and noticed I missed a few of the added periods. I also took the opportunity to clean up some more of the formatting, including removing the potential for extra spaces.

Argenrost reviewed (commented):
Done.
